.rubocop_todo.yml

```diff
@@ -17,7 +17,30 @@ Lint/SuppressedException:
   Exclude:
     - 'lib/tasks/cms.rake'
 
-# Offense count: 9
+# Offense count: 2
+# This cop supports unsafe autocorrection (--autocorrect-all).
+RSpec/BeEq:
```

Comment:
These for example https://www.rubydoc.info/gems/rubocop-rspec/RuboCop/Cop/RSpec/BeEq

Reply:
These updates will be included in my next commit that I push